### PR TITLE
Removed ilsgateway and zipline

### PR DESCRIPTION
**Pull request by orangejenny**
I spoke with product and GS about removing zipline, which ended in early 2017. Zipline turned out to be entangled with ilsgateway, which ended a while ago, so I removed that as well.

### CODE REVIEW

**Review by millerdev — APPROVED** [review]
yay! 🏆 

I guess approval is conditional on REC not using `EMG_AND_REC_SMS_HANDLERS`

File: corehq/toggles.py [hunk 1]
```diff
@@ -1169,13 +1169,6 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_USER]
 )
 
-EMG_AND_REC_SMS_HANDLERS = StaticToggle(
-    'emg_and_rec_sms_handlers',
```
[the remark this answers is not in the document]
Comment (orangejenny) [reply corehq/toggles.py]: I think "rec" here is short for "receipt" - the flag isn't turned on for anyone on prod, and the only references to it were in `custom/ilsgateway/tanzania/handler.py` and `custom/ilsgateway/zipline/reports/zipline_report.py`.

**Comment by czue** [comment]
what stats! :raised_hands: 
image

**Comment by snopoke** [comment]
@orangejenny please can you update https://github.com/dimagi/commcare-hq/blob/aa4c35e015bee31a016a91897c12cd5445acf06c/custom/expired_domains.md

**Comment by orangejenny** [comment]
thanks for the reminder, https://github.com/dimagi/commcare-hq/pull/26680

**Pull request merged**